### ISSUE TITLE
Waiting with presenceOfElementLocated 

### ISSUE THREAD

**Issue by afraca** [php-webdriver/php-webdriver#590]
### What are you trying to achieve? (Expected behavior)
Waiting until an element is available in the DOM. With the reproduction we should be able to continue after 5 seconds.

### What do you get instead? (Actual behavior)
A NoSuchElement exception during the wait.

### How could the issue be reproduced? (Steps to reproduce)
Have a page insert an element with javascript into the dom after 5 seconds. Load the initial page, then do
```php
$driver->wait(10)->until(WebDriverExpectedCondition::presenceOfElementLocated(WebDriverBy::cssSelector('.foo')));
```

### Details

* Php-webdriver version:  1.6
* PHP version: 7.0
* Selenium server version: (directly via ChromeDriver (imitated Laravel Dusk)
* Operating system: MacOS 
* Browser used + version: Chrome 67

### Post scriptum

I think what is supposed to happen is
```php
public static function presenceOfElementLocated(WebDriverBy $by)
    {
        return new static(
            function (WebDriver $driver) use ($by) {
                return $driver->findElement($by);
            }
        );
    }
```

should be 
```php
public static function presenceOfElementLocated(WebDriverBy $by)
    {
        return new static(
            function (WebDriver $driver) use ($by) {
try {
                $driver->findElement($by);

                return true;
            } catch (NoSuchElementException $e) {
                return false;
            }            }
        );
    }
```

**but** I am not terribly familiar with the codebase.

**Comment by afraca** [comment]
I made a minor mistake, the closure should of course return the element, not `true`

**Comment by OndraM** [comment]
This was fixed by #546 .
